kvmagent/kvmagent/plugins/nvram/nvram_common.py

```
@@ -0,0 +1,127 @@
+    is_mounted = False
+    if os.path.exists('/proc/mounts'):
+        with open('/proc/mounts', 'r') as f:
+            is_mounted = any(
+                len(parts) >= 2 and parts[1] == mount_path
+                for parts in (line.split() for line in f)
+            )
+
+    if not is_mounted:
+        ret = bash.bash_r("mount -t ext4 %s %s" % (pipes.quote(source_path), pipes.quote(mount_path)))
+        if ret != 0:
+            raise Exception("failed to mount %s to %s" % (source_path, mount_path))
```

Ruff (0.15.2)

[warning] Create your own exception

(TRY002)